apps/studio/src/components/CoreInterface.vue

```diff
@@ -55,30 +58,33 @@
   import ProgressBar from './editor/ProgressBar.vue'
   import LostConnectionModal from './LostConnectionModal.vue'
   import Vue from 'vue'
-  import { SmartLocalStorage } from '@/common/LocalStorage'
   import RenameDatabaseElementModal from './common/modals/RenameDatabaseElementModal.vue'
-  import { mapGetters } from 'vuex'
+  import { mapGetters, mapActions, mapState } from 'vuex'
+  import _ from "lodash"
+
+  const MAIN_CONTENT_MIN_WIDTH = 200 // in pixels
```

Comment:
Man, wish we had some sort of ini system where we could put stuff like this :eyes:

Reply:
Lol. Should we? I mean we could..